.env.example

```diff
@@ -31,4 +31,5 @@ GOLD_API_BASE_URL=https://gold.jgi.doe.gov/rest/nmdc
 GOLD_API_USERNAME=x
 GOLD_API_PASSWORD=x
 
-NMDC_PORTAL_API_BASE_URL=https://data-dev.microbiomedata.org/
+NEON_API_TOKEN=y
+NEON_API_BASE_URL=https://data.neonscience.org/api/v0
```

Comment:
`NMDC_PORTAL_API_BASE_URL` intentionally removed?

Reply:
Oh wow, great catch! No, I removed it by mistake, apologies 😅